README.md

```diff
@@ -134,6 +138,39 @@ const MyComponent = ({ userId }) => {
 };
 ```
 
+## Listening to events
+
+The core Javascript client will emit different types on events based on what is happening in
```

Comment:
The core JavaScript client emits various types of events depending on internal activities. You can listen to these events by using a hook to access the client and then directly attaching event listeners. Alternatively, if you're using the FlagProvider with a client, you can directly use this client to listen to the events.